products/memcached.md

```diff
@@ -0,0 +1,43 @@
+---
+title: Memcached
+category: db
+iconSlug: memcached
+permalink: /memcached
+versionCommand: memcached -h
+activeSupportColumn: true
+releaseDateColumn: true
+
+identifiers:
+-   purl: pkg:generic/memcached
+-   purl: pkg:deb/ubuntu/memcached
+-   purl: pkg:deb/debian/memcached
+-   purl: pkg:rpm/amzn/memcached
+-   purl: pkg:rpm/redhat/memcached
+-   purl: pkg:rpm/centos/memcached
+-   purl: pkg:docker/library/memcached
+-   repology: memcached
+
+auto:
+-   git: https://github.com/memcached/memcached.git
+
+releases:
+-   releaseCycle: "1.6"
+    releaseDate: 2020-03-09
+    support: true
+    eol: false
+    latest: '1.6.22'
+    latestReleaseDate: 2023-10-09
```

Comment:
Does memcached publish changelogs ?

Reply:
The changelog seems unmaintained https://github.com/memcached/memcached/blob/master/ChangeLog

but they have release notes https://github.com/memcached/memcached/wiki/ReleaseNotes1622